src/main/java/seedu/dukeofbooks/command/BorrowCommand.java

```diff
@@ -4,15 +4,20 @@
 import seedu.dukeofbooks.data.book.BorrowableItem;
 
 public class BorrowCommand extends Command {
-    public static final String COMMAND_WORD = "history";
+    public static final String COMMAND_WORD = "borrow";
     Person person;
     BorrowableItem item;
+    private String title;
     
     public BorrowCommand(Person person, BorrowableItem item) {
         this.person = person;
         this.item = item;
     }
 
+    public BorrowCommand(String title){
+        this.title=title;
+    }
+
```

Comment:
Changes here are redundant, I implemented these in my pr #32 for borrowing books. Can solve the conflict later when merging.